File: 0682/sesión13 - MasterPage+CRUD/SoftInv/SoftInvModel/TiposDocumentosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.Model
{
    public class TiposDocumentosDTO
    {
        private int? tipoDocumentoId;
        private string nombre;

        public TiposDocumentosDTO()
        {
            this.TipoDocumentoId = null;
            this.Nombre = null;
        }

        public TiposDocumentosDTO(int tipoDocumentoId, string nombre)
        {
            this.TipoDocumentoId = tipoDocumentoId;
            this.Nombre = nombre;
        }

        public int? TipoDocumentoId { get => tipoDocumentoId; set => tipoDocumentoId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
    }
}

File: 0682/Sesión 08 - arquitectura/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/TiposDocumentosDTO.java
package pe.edu.pucp.softinv.model;

public class TiposDocumentosDTO {

    private Integer tipoDocumentoId;
    private String nombre;

    public TiposDocumentosDTO() {
        this.tipoDocumentoId = null;
        this.nombre = null;
    }

    public TiposDocumentosDTO(Integer tipoDocumentoId, String nombre) {
        this.tipoDocumentoId = tipoDocumentoId;
        this.nombre = nombre;
    }

    /**
     * @return the tipoDocumentoId
     */
    public Integer getTipoDocumentoId() {
        return tipoDocumentoId;
    }

    /**
     * @param tipoDocumentoId the tipoDocumentoId to set
     */
    public void setTipoDocumentoId(Integer tipoDocumentoId) {
        this.tipoDocumentoId = tipoDocumentoId;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

}
